stowdir/.vim/vimrc

```diff
@@ -632,14 +632,14 @@ let test#strategy = "dispatch"
 " Keybinds
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-" Handle tmux $TERM quirks in vim
-" See: https://stackoverflow.com/a/9674158
-if $TERM =~ '^screen-256color'
-    map <Esc>OH <Home>
-    map! <Esc>OH <Home>
-    map <Esc>OF <End>
-    map! <Esc>OF <End>
-endif
+" " Handle tmux $TERM quirks in vim
+" " See: https://stackoverflow.com/a/9674158
+" if $TERM =~ '^screen-256color'
+"     map <Esc>OH <Home>
+"     map! <Esc>OH <Home>
+"     map <Esc>OF <End>
+"     map! <Esc>OF <End>
+" endif
```

Comment:
Need to test at work - if it works, then remove this commented out cruft. If it doesn't, cry.